demo/api/src/main.ts

```diff
@@ -1,9 +1,9 @@
 import helmet from "helmet";
 
 if (process.env.TRACING == "production") {
-    require("./tracing.production");
+    import("./tracing.production");
```

Comment:
I think this rewrite changes the behaviour because using import like this will cause an promise. and tracing has to be loaded first, I think you need a top-level-await here

Reply:
PS: this has nothing todo with this pr but the import from helm has to be below because tracing has to be the very first import

Reply:
true, i reverted this.